Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+//Floyd-Warshall Algorithm is an algorithm for finding the shortest path between all the pairs of vertices in a weighted graph.
+
+$f    = fopen('php://stdin', 'r');
+
+$line = fgets($f);
+
+$arr  = explode(" ", $line);
+
+$m    = $arr[0]; // first int as m rows 
+$n    = $arr[1]; // second int as n cols
+
+unset($arr[0]);
+unset($arr[1]);
+
+$arr    = array_values($arr);
```

Comment:
please remove the extra space before equal sign

Reply:
@asha15  Done the required changes. Please review.